frontend/src/pages/WhyToTeachPage/WhyToTeachPage.tsx

```diff
@@ -161,9 +162,12 @@ const HowTeachingProgramWorks: React.FC = () => (
         Musíte být <strong>plně způsobilí k právním úkonům</strong>, prokázat{' '}
         <strong>znalost</strong> českého nebo jiného <strong>jazyka</strong>, ve kterém je vedeno
         vyučování, být bezúhonní, <strong>zdravotně způsobilí</strong> a mít{' '}
-        <strong>odbornou kvalifikaci</strong>
-        pro konkrétní přímou pedagogickou činnost. Pro přehledné shrnutí, jak je to s pedagogickou
-        kvalifikací podle zákona o ped. pracovnících navštivte náš rozcestník.{' '}
+        <strong>odbornou kvalifikaci</strong> pro konkrétní přímou pedagogickou činnost. Pro
+        přehledné shrnutí,{' '}
+        <LinkRegular href="https://zacniucit.ceskodigital.net/muzu-ucit#zakon-o-pedagogickych-pracovnicich">
```

Comment:
na ten LinkRegular jde použít taky <Button> komponenta. Je tam prop co se jmenuje něco jako `buttonStyle="link"`. Takhle bys to neměl redirectovaný přes react-router (alespoň myslim, že LinkRegular neni styled Button komponenta, ale styled <a>).